script.js

```diff
@@ -10,3 +10,215 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
+  projectBody.classList.add('project-body');
+  projectBody.append(projectDescription, pDescription, languageUl, projectBtn);
+  projectDescription.classList.add('description');
+  projectDescription.append(
+    canopyText,
+    countImage,
+    backText,
+    countImage2,
+    yearText,
+  );
+
+  projects[i].technologies.forEach((tech, n) => {
+    const languageLi = document.createElement('li');
+    languageLi.classList.add('language-used');
+    languageUl.append(languageLi);
+    languageLi.innerText = projects[i].technologies[n];
+  });
+
+  projectContainer.appendChild(projectDiv);
+
+  projectBtn.classList.add('btn', 'project-link');
+  projectBtn.innerText = 'see project';
+  // reverse  card
+  if (i % 2 !== 0) {
+    projectDiv.classList.add('card-reverse');
+  }
+});
+
+// Menu PopUP
+const modalContainer = document.querySelector('.modal-container');
+const modal = document.createElement('div');
+modalContainer.append(modal);
+modal.classList.add('modal');
+
+modal.innerHTML = `
+<div class="modal-header">
+    <h4 class="modal-heading">Tonic</h4>
+    <h4 class="modal-close">&times;</h4>
+</div>
+<div class="small-description">
+  <p class="canopy modal-describe">CANOPY</p>
+  <img
+    src="./Assets/vectors/Counter.svg"
+    class="count-vector"
+    alt="null"
+  />
+  <p class="modal-describe">BACK END DEV</p>
+  <img
+    src="./Assets/vectors/Counter.svg"
+    class="count-vector"
+    alt="null"
+  />
+  <p class="modal-describe">2015</p>
+</div>
+<img
+  src="./Assets/other-images/Modal-Snapshoot-Portfolio.png"
+  class="modal-img"
+  alt=""
+/>
+<div class="modal-content">
+  <p class="modal-texts">
+    Lorem Ipsum is simply dummy text of the printing and typesetting
+    industry. Lorem Ipsum has been the industry's standard dummy text
+    ever since the 1500s, when an unknown printer took a galley of type
+    and scrambled it 1960s with the releaLorem Ipsum is simply dummy
+  </p>
+  <div class="modal-contact">
+    <ul class="languages modal-ul">
+      <li class="language-used modal-li">HTML</li>
+      <li class="language-used modal-li">CSS</li>
+      <li class="language-used modal-li">Bootstrap</li>
+      <li class="language-used modal-li">gitHub</li>
+      <li class="language-used modal-li">Ruby</li>
+      <li class="language-used modal-li">javaScript</li>
+    </ul>
+    <hr class="modal-hr" />
+    <a href="https://temesgena.github.io/Portfolio-mobile/">
+      <button class="btn modal-btn" type="submit">
+        See live
+        <img
+          src="./Assets/vectors/git-modalicon.svg"
+          class="modal-vectors"
+          alt="git"
+        />
+      </button>
+    </a>
+    <a href="https://temesgena.github.io/Portfolio-mobile/">
+      <button class="btn modal-btn" type="submit">
+        See Source
+        <img
+          src="./Assets/vectors/source-code-icon.svg"
+          class="modal-vectors"
+          alt="source-code"
+        />
+      </button>
+    </a>
+  </div>
+</div>
+`;
```

Comment:
- [ ] The Modal popup data should be populated with the data from the projects array, as for now your using the first project data to generate all the projects popup, kindly fix this so every project generates the proper data. for example instead of using `<h4 class="modal-heading">Tonic</h4>` you can use a loop to iterate over the array and populate the right title